packages/harmony/src/components/comments/ArtistPick/types.ts

```diff
@@ -0,0 +1,5 @@
+export type ArtistPickType = 'picked' | 'liked' | 'both'
+
+export type ArtistPickProps = {
+  type: ArtistPickType
+}
```

Comment:
We've been using the term `pinned` instead of `picked`; prefer that we keep using `pinned` as much as possible (especially now that thats in the db) 

Also it probably makes ergonomic sense to just have `isPinned` and `isLiked` separate flags since the components here are having to do string checks & the comments components will also have to combine into a string anyways (is liked and is pinned will be separate flags anyways)

Reply:
good thought. i'll make them separate 👍